packages/create-cloudflare/templates/svelte/c3.ts

```diff
@@ -67,7 +67,7 @@ const config: TemplateConfig = {
 	configure,
 	transformPackageJson: async () => ({
 		scripts: {
-			"pages:preview": `${npm} run build && wrangler pages dev .svelte-kit/cloudflare`,
+			"pages:preview": `${npm} run build && wrangler pages dev ${await compatDateFlag()} .svelte-kit/cloudflare`,
```

Comment:
the new `no-unused-vars` eslint rule made me notice that here we were still importing `compatDateFlag` but not using it, so I re-added it back here

(PS: no changeset is needed since my changes that remove the flag haven't yet been released)